src/style.css

```diff
@@ -1,23 +1,17 @@
-.blocks {
-  /*display: block;*/
-  /*width: 100%;*/
-}
-
-#block_1 {
+.block {
   content: " ";
```

Comment:
this is redundant
![image](https://user-images.githubusercontent.com/38065883/75157490-e6bbf580-571c-11ea-9b25-6eb4c41ae9e2.png)